clang/lib/Sema/SemaHLSL.cpp

```diff
@@ -245,6 +274,18 @@ void SemaHLSL::CheckEntryPoint(FunctionDecl *FD) {
           << llvm::Triple::getEnvironmentTypeName(ST);
       FD->setInvalidDecl();
     }
+    if (const auto *NT = FD->getAttr<HLSLWaveSizeAttr>()) {
```

Comment:
Why is NT the name here? That doesn't seem to make sense with the context of what you're assigning.

Reply:
Renamed.